### DISCUSSION TITLE
Shelly Plus RGBW PM  2.13 was working now no response

### DISCUSSION THREAD

**Discussion by theHunter2024** [mongoose-os-apps/shelly-homekit#1545] [General]
Shelly Plus RGBW PM used the upgrade path revert to shelly 1.3.3 then upgrade to shelly 1.4.4 then installed Mongoose 2.13, everything was working fine, was very impressed that I did not have any issues, the settings were retained after reboot and all seemed to be fine. Let it run for a couple of days, worked at the first 24hr mark just fine no problems. Then at the 48hr mark no response, rebooted still no response. Light flashing at steady rate was on dim when working properly. Tried to reset by holding button for 10 seconds and it has a series of fast flashes then goes back to the steady flashing. I have checked my connected devices on my network and the shelly is listed as offline, checked bluetooth and no devices found. Home just says no response, tried deleting and setting up a new accessory but it is unable to find anything. 

What is my next step or has this unit failed?

**Comment by markirb** [comment]
I do not think it has failed. It would be very unexpected.

I think it rebooted and is in factory defaults. Are you sure you see no WiFi network?

I think it has to do with
https://github.com/mongoose-os-apps/shelly-homekit/issues/1518

I am still not sure the pinning for the reset is correct.

**Comment by saubauerfred** [comment]
Hello, 
I did shelly plus 1 for garage update. At homekit web page on swap imput pressed. Since then wifi is off and doesn't come up anymore means with homekit flashed shelly. Not reachable via wifi. Is there a possibility? Thank you

**Reply by markirb** [reply]
This is completely unrelated to this topic.

However: Do you have an add-on? Swapped inputs only makes sense then.
I fear that it could lead indeed to a crash on +1 without add-on. 

A factory reset via button should help.

**Comment by saubauerfred** [comment]
Thanks for the answer
Unfortunately, I can't reset it via the reset button

**Reply by markirb** [reply]
Why not? What does the  blinking LED say?

**Comment by saubauerfred** [comment]
I don't see Led. But somehow I managed it with plug in reset button etc. etc. that I get access via 192.168.33.1 and then I saw wifi was turned off etc. 

thank you

**Reply by markirb** [reply]
ok. 

curious what you were trying with the swap sensors?

next version will avoid this illegal state which causes a crash...

**Reply by mikecitt** [reply]
@saubauerfred how did you reset it? What do you mean by “plug in reset button”?

**Reply by saubauerfred** [reply]
> @saubauerfredWie hast du es zurückgesetzt? Was meinen Sie mit "Einstecktaste zum Zurücksetzen"?
Hello, I can't say exactly how I could reset the shelly. I pressed the reset button for a long time. And maybe that helped

**Comment by saubauerfred** [comment]
I thought I could change the open condition with it. Was a mistake. Why this happens is no idea. 
I even set shelly at the 2 garages. Both didn't work anymore. Then I have 2 new programs.

**Comment by theHunter2024** [comment]
I was able to connect to the internal wifi server and you are correct it did loose all of the settings like in 2.12.2. Any update for a fix for this or is this going to be an ongoing issue due to a different architecture on the board?

**Reply by markirb** [reply]
ok good.

I mean it has different Pinout and we have to figure that out. once that is correct there should be no issue.

The information from https://github.com/mongoose-os-apps/shelly-homekit/discussions/1502 sounded like reset was right in 2.12, then some people had problems and I changed it again.

The big problem is I cannot verify the whole thing myself.
https://github.com/mongoose-os-apps/shelly-homekit/issues/1518
